README.md

```diff
@@ -123,46 +125,177 @@ Additionally, you'll need to copy debug symbols for debugging and crash reportin
+// as result you get users JWT, it looks like this: "eyJraWQiOiI3MGI0NDd...RvMK7p7Ak"
+// you can provide users with JWT at registration or authorization steps
+// Send a JWT to client-side
+let jwtString = aliceJwt.stringRepresentation()
+```
 
-// retrieve key data
-let retrievedEntry = try! keychainStorage.retrieveEntry(withName: identity)
+For this subsection we've created a sample backend that demonstrates how you can set up your backend to generate the JWTs. To set up and run the sample backend locally, head over to your GitHub repo of choice:
+
+[Node.js](https://github.com/VirgilSecurity/sample-backend-nodejs) | [Golang](https://github.com/VirgilSecurity/sample-backend-go) | [PHP](https://github.com/VirgilSecurity/sample-backend-php) | [Java](https://github.com/VirgilSecurity/sample-backend-java) | [Python](https://github.com/VirgilSecurity/virgil-sdk-python/tree/master#sample-backend-for-jwt-generation)
+ and follow the instructions in README.
+ 
+### Set up Card Verifier
+
+Virgil Card Verifier helps you automatically verify signatures of a user's Card, for example when you get a Card from Virgil Cards Service.
+
+By default, `VirgilCardVerifier` verifies only two signatures - those of a Card owner and Virgil Cards Service.
+
+Set up `VirgilCardVerifier` with the following lines of code:
+
+```swift
+import VirgilSDK
+import VirgilCrypto
+
+// initialize Crypto library
+let cardCrypto = VirgilCardCrypto()
+
+let yourBackendVerifierCredentials =
+    VerifierCredentials(signer: "YOUR_BACKEND",
+                        publicKey: Data(base64Encoded: publicKeyStr)!)
+
+let yourBackendWhitelist =
+    try! Whitelist(verifiersCredentials: [yourBackendVerifierCredentials])
+
+let cardVerifier = VirgilCardVerifier(cardCrypto: cardCrypto,
+                                      whitelists: [yourBackendWhitelist])
 
-// import key from Data
-let privateKey = try! exporter.importPrivateKey(from: retrievedEntry.data)
 ```
 
-#### Generate and publish user's Cards with Public Keys inside on Cards Service
-Use the following lines of code to create and publish a user's Card with Public Key inside on Virgil Cards Service:
+### Set up Card Manager
+
+This subsection shows how to set up a Card Manager module to help you manage users' public keys.
+
+With Card Manager you can:
+- specify an access Token (JWT) Provider.
+- specify a Card Verifier used to verify signatures of your users, your App Server, Virgil Services (optional).
+
+Use the following lines of code to set up the Card Manager:
+
+```swift
+// initialize cardManager and specify accessTokenProvider, cardVerifier
+let cardManagerParams = CardManagerParams(cardCrypto: cardCrypto,
+                                          accessTokenProvider: accessTokenProvider,
+                                          cardVerifier: cardVerifier)
+
+let cardManager = CardManager(params: cardManagerParams)
+```
+
+### Set up Key Storage for private keys
```

Comment:
The whole part `Set up Key Storage for private keys` should be deleted from all our resources (website, readme etc.). It's super old.